src/Runner.jl

```diff
@@ -986,7 +988,7 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
         "CARGO_HOME" => "/opt/$(host_target)",
         "RUSTUP_HOME" => "/opt/$(host_target)",
         # TODO: we'll need a way to parameterize this toolchain number
-        "RUSTUP_TOOLCHAIN" => "1.43.0-$(map_rust_target(host_platform))",
+        "RUSTUP_TOOLCHAIN" => "1.56.1-$(map_rust_target(host_platform))",
```

Comment:
Memo to self: the Rust and Go variables shouldn't be set if their compilers aren't requested.